docs/2021-ga-release-blog-post.md

```diff
@@ -0,0 +1,58 @@
+We're excited to announce the new GA release for Javascript SDKs. These SDKs have several new features that help our customers to have a great experience in using the Azure Services. Some of the important SDKs that have been released are:
+
+1. [Compute](https://www.npmjs.com/package/@azure/arm-compute/v/17.0.0)
+2. [Network](https://www.npmjs.com/package/@azure/arm-network/v/26.0.0)
+3. [Storage](https://www.npmjs.com/package/@azure/arm-storage/v/17.0.0)
+4. [Sql](https://www.npmjs.com/package/@azure/arm-sql/v/9.0.0)
+5. [Resources](https://www.npmjs.com/package/@azure/arm-resources/v/5.0.0)
+6. [Policy](https://www.npmjs.com/package/@azure/arm-policy/v/5.0.0)
+7. [Managed applications](https://www.npmjs.com/package/@azure/arm-managedapplications/v/2.0.0)
+8. [Links](https://www.npmjs.com/package/@azure/arm-links/v/2.0.0)
+9. [Locks](https://www.npmjs.com/package/@azure/arm-locks/v/2.0.0)
+10. [Features](https://www.npmjs.com/package/@azure/arm-features/v/3.0.0)
+
+Each of these SDKs follows the new Azure SDK guidelines. All these SDKs have been generated using the latest version of the new [Autorest Typescript SDK Generator](https://www.npmjs.com/package/@autorest/typescript/v/6.0.0-beta.15). These libraries provide new authentication mechanisms, uses Promises, etc. We encourage you to try out the libraries and provide your feedback.
```

Comment:
Shall we add a few sentenses to describe those guidelines?

Reply:
Added link to the guidelines